lampod/src/ln/inventory_manager.rs

```diff
@@ -33,11 +33,32 @@ impl InventoryHandler for LampoInventoryManager {
         match event {
             InventoryCommand::GetNodeInfo(chan) => {
                 let chain = self.channel_manager.conf.network.to_string();
+                let alias = self
+                    .channel_manager
+                    .conf
+                    .alias
+                    .as_ref()
+                    .map(|alias| alias.to_string());
```

Comment:
nit: If you were reading this code instead of writing it, would you think it was insane?